thangs_fetcher.py

```diff
@@ -24,10 +25,15 @@
                        )
 from bpy_extras.object_utils import AddObjectHelper, object_data_add
 from bpy.app.handlers import persistent
+import socket
 
 amplitude = ThangsEvents()
 amplitude.deviceId = socket.gethostname().split(".")[0]
 
+config_obj = configparser.ConfigParser()
+config_path = os.path.join(os.path.dirname(__file__), 'prod_config.ini')
+config_obj.read(config_path)
+thangs_config = config_obj['thangs']

```

Comment:
Can we take this little snippet, move it into its own `config.py`, and then import `thangs_config` from `config.py` in both here and the events class?  It'll take out that little bit of duplication that we have in those classes